ateam_geometry/include/ateam_geometry/variant_do_intersect.hpp

```diff
@@ -61,15 +83,13 @@ bool variantDoIntersect(const ObjA & object_a, const AnyShape & object_b)
  * @return true The two geometry objects intersect
  * @return false The two geometry objects do not intersect
  */
-template<>
```

Comment:
What motivated removing this?

Reply:
I thought I added this by accident previously and didn't want to add anything extra that wasn't need... I'll add it back in!